### ISSUE TITLE
Are there any plans for extensive built-in mapping?

### ISSUE THREAD

**Issue by xBitsoft** [mfosse/JoyCon-Driver#77]
By extensive mapping I mean the ability to map to a virtual x360 controller, keyboard and/or mouse, with or without profiles.

**Comment by mfosse** [comment]
Yeah, I want to do this, but I don't have the time, maybe after finals I'll convert the program to Qt and make it happen

**Comment by mfosse** [comment]
(finals end May 1st)
